examples/Voter.cpp

```diff
@@ -61,12 +67,24 @@ int64_t numSPCalls;
 int64_t minAllowedOutstanding;
 int64_t maxAllowedOutstanding;
 
+#ifdef _MSC_VER
+// this version may not be correct
```

Comment:
this comment needs to be cleaned up. Earlier when had looked at it, it seemed incorrect as it does not take into account epoch time similar to gettimeofday(). But later after looking how this routine is used, taking into account epoch is not needed. Will clean up the comment. Thanks